e2e_tests/data/coco/to/output.json

```diff
@@ -85,13 +85,13 @@
   ],
```

Comment:
e2e tests broke again because of the ordering. Will put in a ticket for a fix on the coco exporter to always sort before saving.

Reply:
and it reverted itself and had to be redone.